.github/PULL_REQUEST_TEMPLATE/adr-template.md

```diff
@@ -0,0 +1,77 @@
+---
+name: ADR Template
```

Comment:
Since I was not aware of the ADR format, I wonder if we could use more commonly used terminology here? RFC is the one I find most common and I know @sheplu even mentioned that on our call. That said, "proposal" would also make sense to me, and I think is more in the loose spirit of what these can be.

Reply:
RFC sounds great! :)

Reply:
an RFC is something different and I think we should prefer the semantically correct term for what this represents.  `RFC !== ADR`